client/dom/genesetEdit.ts

```diff
@@ -251,13 +251,18 @@ export function showGenesetEdit(arg: showGenesetEditArg) {
 	}
 
 	function renderGenes() {
+		const orderedGenes = geneList.sort((a, b) => {
```

Comment:
when `gene.mutationStat[]` is present, please do not sort alphabetically as genes are already ranked on the server according to stat

also mutationStat is not declared in the type def

Reply:
Thanks for pointing this out. Done.